scripts/test_publish

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh -e
+
+VERSION_FILE="deta/__init__.py"
+
+if [ -d 'venv' ] ; then
+    PREFIX="venv/bin/"
+else
+    PREFIX=""
+fi
+
+if [ ! -z "$GITHUB_ACTIONS" ]; then
+  git config --local user.email "action@github.com"
+  git config --local user.name "GitHub Action"
```

Comment:
is this recommended? don't we need to have our own details?

Reply:
We can do that, that just seems to make sure that the tags and releases are attributed to the Github bot!

Reply:
@abdelhai is there a specific username/email combo you'd like associated with those actions? :)